### PR TITLE
PDF cleanup: Fig 36

### PR DESCRIPTION
**Pull request by williamdemeo**
# Description

Address the following items of issue #146 concerning Figure 36.

- [ ] We shouldn't show `SpecProperty` stuff, `sp-∘` in this case.
- [X]  The pattern-matching trick in `acceptedBy` to expose the fact that the number isn't 0 (i.e. `x@(suc _)`) is black magic. We shouldn't do this in the figure.

# Checklist

- [X] Commit sequence broadly makes sense and commits have useful messages
- [X] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [ ] Self-reviewed the diff

### CODE REVIEW

**Comment by williamdemeo** [comment]
I'm not sure how to present `getStakeDist` without exposing the `SpecProperty` stuff (`sp-∘`).  @WhatisRT suggestions?

File: src/Ledger/Ratify.lagda [hunk 1]
```diff
@@ -201,16 +201,16 @@ mostStakeDRepDist-∅ {dist} = suc (Σᵐᵛ[ x ← dist ᶠᵐ ] x) , Propertie
         <⟨ v>sum ⟩
       v ∎
 
-∃topNDRepDist : ∀ {n dist} → lengthˢ (dist ˢ) ≥ n → n > 0
-                → ∃[ c ] lengthˢ (mostStakeDRepDist dist c ˢ) ≥ n
+∃topNDRepDist : ∀ {n dist} → n ≤ lengthˢ (dist ˢ) → n > 0
```
Comment (williamdemeo) [inline src/Ledger/Ratify.lagda:204]: Reserve `≥` for rationals in this module.

(quick and dirty way to use `≥` in place of `ℚ.≥` below)

File: src/Ledger/Ratify.lagda [hunk 2]
```diff
@@ -364,22 +364,40 @@ abstract
...
+    redStakeDistr : StakeDistrs
+    redStakeDistr = restrictedDists coinThreshold rankThreshold stakeDistrs
+
+    t : ℚ
+    t = maybe id ℚ.0ℚ $ threshold pparams (proj₂ <$> cc) action role
+
+    stakeFilter : Coin → Set
+    stakeFilter zero = t ≡ ℚ.0ℚ
+    stakeFilter (suc x) = (acceptedStake role cc' redStakeDistr votes')ᶻ / suc x ≥ t
+
```
Comment (williamdemeo) [inline src/Ledger/Ratify.lagda:397]: The proof of `acceptedBy` is now longer but hopefully much simpler and easier for non-experts to follow and maybe even comprehend.

File: src/Ledger/Ratify.lagda [hunk 3]
```diff
@@ -364,22 +364,40 @@ abstract
...
+  accepted : RatifyEnv → EnactState → GovActionState → Set
+  accepted Γ es gs = acceptedBy Γ es gs CC ∧ acceptedBy Γ es gs DRep ∧ acceptedBy Γ es gs SPO
+
```
Comment (williamdemeo) [inline src/Ledger/Ratify.lagda:400]: We might simplify the presentation of `acceptedBy` and `accepted` further with a (pdf-hidden) `module _` directive with parameters `Γ : RatifyEnv`, `es : EnactState`, and `gs : GovActionState`.

File: src/Ledger/Ratify.lagda (hunk 1 above)
Comment (WhatisRT) [reply src/Ledger/Ratify.lagda:204]: Now that you worked so hard on the typeclasses, you should just make an instance for `ℚ`!

File: src/Ledger/Ratify.lagda (hunk 3 above)
Comment (WhatisRT) [reply src/Ledger/Ratify.lagda:400]: Or just put `acceptedBy' = acceptedBy Γ es gs` in a `where` block

**Review by WhatisRT — CHANGES_REQUESTED** [review]
I didn't really like that this is longer and it only seemed marginally better, so I decided to help you out, see #250. There's still some more stuff to do though.

**Comment by williamdemeo** [comment]
@WhatisRT has a better approach (#250) addressing most of the stuff here.

**Pull request closed without merging**